packages/visualisation/src/components/HoverInfoBox/index.js

```diff
@@ -46,7 +46,10 @@ const CarInfo = ({ data }) => {
         <Paragraph>
           Köat: {data.queue || 0} {label}
         </Paragraph>
-        <Paragraph>Co2: {data.co2 || 0} kg</Paragraph>
+        <Paragraph>Co2: {Math.ceil(10 * data.co2) / 10 || 0} kg</Paragraph>
+        <Paragraph>
+          Körsträcka: {Math.ceil(10 * data.distance) / 10 || 0} km
```

Comment:
I added the total distance traveled to the hover info. I think it can be interesting to see and it makes it easier to validate the co2 emissions.